android/src/main/java/com/swmansion/rnscreens/Screen.kt

```diff
@@ -345,6 +347,35 @@ class Screen(
         this.headerConfig?.toolbar?.importantForAccessibility = mode
     }
 
+    // Accepts one of 3 focusable flags and one of 3 descendantFocusability flags.
+    // https://developer.android.com/reference/android/view/View#setFocusable(int)
+    // https://developer.android.com/reference/android/view/ViewGroup#setDescendantFocusability(int)
+    @RequiresApi(Build.VERSION_CODES.O)
+    fun changeFocusability(
+        focusable: Int,
+        descendantFocusability: Int,
+    ) {
+        this.focusable = focusable
+        this.headerConfig?.toolbar?.focusable = focusable
+
+        this.descendantFocusability = descendantFocusability
+        this.headerConfig?.toolbar?.descendantFocusability = descendantFocusability
+    }
+
+    // Accepts boolean for focusability and one of 3 descendantFocusability flags.
+    // This method is provided for compatibility reasons only (SDK_API <= 25).
+    // https://developer.android.com/reference/android/view/ViewGroup#setDescendantFocusability(int)
+    fun changeFocusabilityCompat(
+        focusable: Boolean,
+        descendantFocusability: Int,
+    ) {
+        this.setFocusable(focusable)
+        this.headerConfig?.toolbar?.setFocusable(focusable)
+
+        this.descendantFocusability = descendantFocusability
+        this.headerConfig?.toolbar?.descendantFocusability = descendantFocusability
```

Comment:
nit: it repeats with the prev method, so you may consider extracting the logic for descendant focusability